### DISCUSSION TITLE
MRZ extraction in passport using ppocr

### DISCUSSION THREAD

**Discussion by pbnewron** [PaddlePaddle/PaddleOCR#8852] [General]
Hi, 
did anyone tried ppocr for MRZ extraction in passport? I was working with rec_v3 models and ocr performance is not very good compared to Tesserect. Does paddleocr out of box works for OCR-B font ( font used in MRZ section of passports)?
Can anyone suggest a good finetuning document or notebook, the one given in documentation dint help much?

regards
Prashant

**Comment by lishuoren** [comment]
Hello, I have encountered the same problem as you, did you solve the problem?

**Comment by Frosty-GTRIIP** [comment]
Hi there, I'm facing the same issue. It looks like the rec_v3 models occasionally fail to detect MRZ boxes. To address this, I'm currently retraining it as a detection model repeatedly. Have you found a solution to this problem?
